src/internal/connector/graph/middleware.go

```diff
@@ -122,42 +124,61 @@ func (mw *LoggingMiddleware) Intercept(
 		return resp, err
 	}
 
-	ctx = clues.Add(ctx, "status", resp.Status, "statusCode", resp.StatusCode)
-	log := logger.Ctx(ctx)
+	ctx = clues.Add(
+		ctx,
+		"status", resp.Status,
+		"statusCode", resp.StatusCode,
+		"content_len", resp.ContentLength)
 
-	// Return immediately if the response is good (2xx).
-	// If api logging is toggled, log a body-less dump of the request/resp.
-	if (resp.StatusCode / 100) == 2 {
-		if logger.DebugAPIFV || os.Getenv(log2xxGraphRequestsEnvKey) != "" {
-			log.Debugw("2xx graph api resp", "response", getRespDump(ctx, resp, os.Getenv(log2xxGraphResponseEnvKey) != ""))
-		}
+	var (
+		log       = logger.Ctx(ctx)
+		respClass = resp.StatusCode / 100
```

Comment:
nit: I've seen this statusCode / 100 technique being used in several places in the code to find resp class. It's clever! 
Can we move this to a small helper? Doing raw divisions in the code can be prone to bugs.

Reply:
I think a helper is overbearing in this case, since it's pretty well localized code with clear inputs and outputs.  But if we start to see it pop up in multiple packages (say, if we generate clients outside of graph), that'll be a good time to centralize.